WordPress/Classes/ViewRelated/Media/StockPhotos/StockPhotosStrings.swift

```diff
@@ -6,11 +6,7 @@ extension String {
     }
 
     static var otherApps: String {
-        return NSLocalizedString("Other Apps", comment: "Menu option used for adding media from other applications.")
-    }
-
-    static var allFiles: String {
-        return NSLocalizedString("Choose from device", comment: "Menu option used for adding media from other applications.")
+        return NSLocalizedString("Other apps", comment: "Menu option used for adding media from other applications.")
```

Comment:
I noticed that the text changed from `Other Apps` to `Other apps` (note the downcase of `apps`), is it correct?

Reply:
Thanks for spotting that. I will utilize the upper case variant :)